data/json/items/comestibles/bread.json

```diff
@@ -296,12 +296,25 @@
     "calories": 74,
     "description": "A sweet bread, like cake.",
     "price": "272 cent",
-    "price_postapoc": "4 USD",
+    "price_postapoc": "2 USD",
     "material": [ "wheat" ],
     "volume": "48 ml",
     "charges": 10,
     "flags": [ "EATEN_HOT", "EATEN_COLD" ],
     "fun": 3,
     "vitamins": [ [ "calcium", 2 ], [ "iron", 6 ] ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "bread_canned",
+    "name": { "str_sp": "canned bread" },
+    "copy-from": "brown_bread",
+    "spoils_in": "730 days",
+    "container": "can_medium",
+    "description": "It reads: New England's Finest.",
```

Comment:
Unless there are better ideas, this is the description. It's specific to B&M though...